chart/templates/import-api/configmap.yaml

```diff
@@ -46,8 +46,12 @@ data:
   IMPORT_PROVIDER: {{ .Values.appConfigValues.storageProvider | quote }}
   IMPORT_BUCKET: {{ .Values.appConfigValues.importBucket | quote }}
   {{- if eq .Values.appConfigValues.storageProvider "gcp" }}
-  IMPORT_PROJECTID: {{ include "carto.gcpBucketsProjectId" . | quote }}
+    {{- if .Values.appConfigValues.gcpBucketsProjectId }}
+  IMPORT_PROJECTID: {{ .Values.appConfigValues.gcpBucketsProjectId | quote }}
```

Comment:
Take care here because `IMPORT_PROJECTID` has a value regardless of whether    `.Values.appConfigValues.gcpBucketsProjectId` it's defined or not

https://github.com/CartoDB/carto-selfhosted-helm/blob/94209435909c1c2a1628d41aa1f2c49960d98bc4/chart/templates/_helpers.tpl#L136-L141

Reply:
I have consulted the values in production and the variable is not used so it only seems to be necessary for the bucket customs.

Any way, I didn't test the change. I'm trying to convince Moises to test that for me :see_no_evil:

Reply:
👍  In that case we should remove the `carto.gcpBucketsProjectId` from `_helpers.tpl`

Reply:
Already done